python/xorbits/_mars/tensor/datasource/tests/test_datasource_execution.py

```diff
@@ -822,6 +823,37 @@ def test_triu_execution(setup):
     assert res.flags["F_CONTIGUOUS"] == expected.flags["F_CONTIGUOUS"]
 
 
+def test_tri_execution(setup):
```

Comment:
Could you add some tests for `chunk_size` parameter?

Reply:
Added.